modules/ivc_champva/app/controllers/ivc_champva/v1/uploads_controller.rb

```diff
@@ -73,7 +73,9 @@ def get_attachment_ids_and_form(parsed_form_data)
 
         applicants_count = parsed_form_data[applicant_key]&.count.to_i
         total_applicants_count = applicants_count.to_f / additional_pdf_count
-        applicant_rounded_number = total_applicants_count.ceil
+        # Must always be at least 1, so that `attachment_ids` still contains the
+        # `form_id` even on forms that don't have an `applicants` array (e.g. FMP2)
+        applicant_rounded_number = total_applicants_count.ceil.zero? ? 1 : total_applicants_count.ceil
```

Comment:
We technically always have one applicant, whether that array is present or not. This fixes that oversight, which consequently fixes an off-by-one error when tallying supporting documents later in the form submit process.